pyro/infer/tracegraph_elbo.py

```diff
@@ -279,6 +275,7 @@ def _loss_and_grads_particle(self, weight, model_trace, guide_trace):
         guide_trace.log_pdf()
         model_trace.compute_batch_log_pdf(site_filter=lambda name, site: name in model_vec_md_nodes)
         model_trace.log_pdf()
+        guide_trace.compute_score_parts()
```

Comment:
`.compute_score_parts()` does all the work done by `.log_pdf()` above, so you can remove the `.log_pdf()` call, or just rename it to `.compute_score_parts()`.

Reply:
yup good point done